templates/create-frontend-plugin/docs/features/rbac.md

```diff
@@ -0,0 +1,129 @@
+## Guide to Enforcing Access Controls in your Front-end Plugin
```

Comment:
```suggestion
## Guide to Enforcing Access Controls in your Frontend Plugin
```

Please check all docs for the frontend and plugin and use the Capalized version or not. The need headline use plugin for example